client/src/components/FeedbackForm/FeedbackForm.js

```diff
@@ -0,0 +1,109 @@
+import React, { useMemo } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import PropTypes from 'prop-types';
+import { Formik, Form } from 'formik';
+import { Alert, Button, FormGroup, Spinner } from 'reactstrap';
+import { FormField } from '_common';
+import * as Yup from 'yup';
+import './FeedbackForm.module.scss';
+
+const formSchema = Yup.object().shape({
+  subject: Yup.string().required('Required'),
+  name: Yup.string().required('Required'),
+  email: Yup.string()
+    .email('Invalid email')
+    .required('Required'),
+  problem_description: Yup.string().required('Required')
+});
+
+const FeedbackForm = ({ authenticatedUser }) => {
+  const dispatch = useDispatch();
+  const creating = useSelector(state => state.ticketCreate.creating);
+  const creatingError = useSelector(state => state.ticketCreate.creatingError);
+  const creatingErrorMessage = useSelector(
+    state => state.ticketCreate.creatingErrorMessage
+  );
+
+  const defaultValues = useMemo(
+    () => ({
+      subject: 'Feedback for the Frontera Portal',
+      problem_description: '',
+      name: authenticatedUser
```

Comment:
As we are assuming that the user will always be authenticated when leaving feedback, you might want to consider the following:

- `FeedbackForm`  -> making authenticatedUser a required property instead of an optional property.  Alternatively, consider moving  const authenticatedUser = useSelector(state => state.authenticatedUser.user); from FeedbackModal into FeedbackForm
- `FeedbackForm` -> show spinner while we wait for authenticatedUser (or error message if that request fails)
- then you can simplify the `email: authenticatedUser ? authenticatedUser.email : ''" `  (and the similar ` name) to just `email: authenticatedUser.email` which would be good because the related input fields are disabled anyways because we are assuming an authenticated user

Reply:
Moving moving const authenticatedUser seems like a better idea, thanks @nathanfranklin!

Reply:
Hi @nathanfranklin, like creating a spinner on this line?
```
useSelector(
    state => state.ticketCreate.creatingErrorMessage
  );
```